UIKitTemplate/UIKitTemplate/CustomButton.swift

```diff
@@ -0,0 +1,21 @@
+// CustomButton.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// кастомная кнопка для 2 го вью контроллера
```

Comment:
```suggestion
/// кастомная кнопка для 2 го вью контроллера
```
только для второго? а в третьем нельзя ей пользоваться?